corehq/apps/domain/templates/domain/email/domain_transfer_confirm.html

```diff
@@ -0,0 +1,8 @@
+Initiator: {{ from_username }}
+Project Space: {{ domain }}
+Acceptor: {{ to_username }}
+
+Please validate that the transfer is valid and email both parties notifying them that the transfer is complete.
```

Comment:
Fair enough. I suggested "verify" instead of "validate" because the sentence uses "valid" four words later. Sorry for nitpicking style on something you didn't even write.